src/main.rs

```diff
@@ -43,6 +44,10 @@ struct Cli {
     /// timestamped, e.g. through journald.
     #[structopt(long)]
     no_timestamp: bool,
+    /// Compose the ping behaviour together with the rendezvous behaviour in
+    /// case a rendezvous server with Ping is required. This feature will be removed once https://github.com/libp2p/rust-libp2p/issues/2109 is fixed.
```

Comment:
Are you aware that is is going to show up in `--help`?

Reply:
yes